bigbluebutton-html5/private/config/settings.yml

```diff
@@ -22,6 +22,7 @@ public:
     enableLimitOfViewersInWebcam: false
     enableTalkingIndicator: true
     mirrorOwnWebcam: false
+    maxUsersForConfirmation: 8
```

Comment:
Consider including EndMeeting in the name - something like `maxListSizeEndMeetingWarning` (even though it's kind of long)
At first I thought you were introducing a constant to control how many active users would justify a warning (when ending a meeting) but sounds like you're only limiting the visuals - the max length of the list. Am I understanding this correctly?

Reply:
Yes, exactly. I wanted it to be a big longer, but was scared it would end up in camel casing hell. I'll rework it.